### PR TITLE
ENH: add title to pycortex output.

### PR DESCRIPTION
**Pull request by bcipolli**
Fix for #398 . Enabled by pycortex change https://github.com/gallantlab/pycortex/pull/141 .

I still don't have a working test environment, but fix is simple enough that I'm hoping this just works... and that someone can try this easily!

### CODE REVIEW

**Comment by bcipolli** [comment]
Note: my editor auto-lints... so lots of changes are simply trailing whitespace removal.

**Comment by chrisfilo** [comment]
Thanks! For this to work smoothly we need to fix a particular version of pycortex in Dockerfile and requerements.txt. Since pycortex does not release very often we should just go with a commit hash.

**Comment by bcipolli** [comment]
@chrisfilo added pointer to pycortex commit d064ec5deef5712e550034b36fd83549d3068462, to `Dockerfile` and `requirements.txt`. Only tested that the zip URL is downloadable (it is).

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ RUN pip install numpy \
     cython
 RUN pip install -v scipy
 RUN pip install scikit-learn pandas h5py matplotlib
-RUN pip install https://github.com/gallantlab/pycortex/archive/master.zip --egg
+RUN pip install https://github.com/bcipolli/NeuroVault/archive/d064ec5deef5712e550034b36fd83549d3068462.zip --egg
```
Comment (chrisfilo) [inline Dockerfile]: `bcipolli/NeuroVault` not `gallantlab/pycortex`?

Comment (chrisfilo) [inline Dockerfile]: Also the hash is different than in the requirements.txt

Comment (bcipolli) [inline Dockerfile]: My goodness, embarrassing. Will fix now. thx

**Comment by chrisfilo** [comment]
Thanks for sorting this out!

**Comment by chrisfilo** [comment]
I take it back - your PR does not change the title. I check on local machine and the production server. I even rebuilt neurovault_fs image locally - all with the same result: "Brain" title for bot individual image or collection 3D view. I think you will have to finally install the dev environment and test your fix. Did it work outside of NeuroVault?

**Comment by bcipolli** [comment]
Sure thing. I did test it in the pycortex branch. I am setting up the neurovault+pycortex Docker test environment now.

**Comment by bcipolli** [comment]
@chrisfilo What are the steps to launch the `neurovault_fs` image? I ran:
```
docker-machine create --driver virtualbox nv 
docker-machine start nv
eval "$(docker-machine env nv)
docker-compose up
docker build -t neurovault/neurovault_fs -f fs_docker/Dockerfile .
docker-compose stop
docker-compose up
```

Still, nothing in `/opt`, so "3D view" gives me a freesurfer error. (Checked using `docker run -i -t ubuntu /bin/bash`).

**Pull request merged**